app/assets/javascripts/logs.coffee

```diff
@@ -70,6 +72,19 @@ class Log
 
     updateTotalHours()
 
+    projectChangeListener = ->
+      $("select[id$='_project_id']").on "change", distinctProjects
+      $("select[id$='_project_id']").each distinctProjects # Doesn't work yet.
+
+    distinctProjects = ->
+      console.log("hit it")
+      $("select[id$='_project_id']").children().removeAttr 'disabled'
+      $("select[id$='_project_id']").each ->
+        if $(this).val() != ''
+          $("select[id$='_project_id']").not($(this)).find('option[value=' + $(this).val() + ']').attr 'disabled', 'disabled'
+
+    setTimeout projectChangeListener, 2000
```

Comment:
What's the `setTimeout` for?

Reply:
We used it in SQSS. I assumed it was to start the listener after the page loads.